mouse/record.py

```diff
@@ -10,6 +10,17 @@
 import os
 
 
+class Trigger(object):
```

Comment:
Add docstring explaining what this class does. Look at `ExampleClass` in these docs to see how to document the class and methods ( http://sphinxcontrib-napoleon.readthedocs.org/en/latest/example_google.html )